docs/reference/images.md

```diff
@@ -270,3 +274,34 @@ $ sudo ig image list
 REPOSITORY                     TAG                           DIGEST       CREATED
 trace_open                     latest                        19ea8377298f 30 minutes ago
 ```
+
+#### `inspect`
+
+Inspect the given gadget image.
```

Comment:
```suggestion
Inspect the given local gadget image.
```

AFAIU, inspect does not automatically pull an image if it does not exist locally.